src/main/resources/application.yml

```diff
@@ -42,7 +42,6 @@ queueconfig:
   outbound-field-queue: Action.Field
   outbound-exchange: action-outbound-exchange
   action-case-exchange: action-case-exchange
-  action-case-queue: action.events
```

Comment:
Why has this queue been removed?

Reply:
in the production code, we publish to an exchange, not a queue

Reply:
Does it still need to be in the definitions.json file then?